Resources/Textures/Objects/Consumable/Food/Baked/pie.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24, mime tart slice and banana cream pie slice from rosysyntax under under CC BY-SA 4.0",
```

Comment:
Please don’t remove the part where the other person’s sprite is under a different copyright, otherwise seems good to me i think.

Reply:
see the comment above, the author of that sprite agreed to change the licence so that we don't have to move the file into a separate .rsi folder

Reply:
yea lol nevermind then